config/locales/es.yml

```diff
@@ -49,6 +49,7 @@ es:
     attend: "Apuntarse"
     not_attend: "Cancelar"
     login_before_attend: "Por favor inicia sesión antes"
+    attend_to_view_attendees_information: "Información solo visible para los participantes"
```

Comment:
@mediafinger is this Spanish translation ok?

Reply:
👌